crates/oxc_linter/Cargo.toml

```diff
@@ -28,6 +28,7 @@ phf         = { workspace = true, features = ["macros"] }
 num-traits  = { workspace = true }
 
 rust-lapper = "1.1.0"
+prettytable-rs = "0.10.0"
```

Comment:
We can probably roll our own basic implementation if we don't want a dependency added here